src/js/steps/letters/generateOverlay.js

```diff
@@ -168,16 +168,16 @@ module.exports = function (options, $events) {
        */
       function revertCharsToOriginal(callback) {
         data.combinedLetters
-          .map((letter, index) => ({ ...letter, index }))
+          .map((letter, index) => $.merge(letter, { index }))
           .filter(letter => letter.changed === true)
-          .reduce((arr, letter) => {
-            return arr.concat(
-              letter.characters
-                .map(character => ({ ...character, pageIndex: letter.index }))
-                .filter(character => character.character === letter.default_character)
-            );
+          .reduce(function (arr, letter) {
+            var characters = letter.characters
+              .map(character => $.merge(character, { pageIndex: letter.index }))
+              .filter(character => character.character === letter.default_character);
+
+            return [...arr, ...characters];
```

Comment:
It's straight from the documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator